posts/Fabio/2018-10-01-hibernate-ogm-5-4-CR1-released.adoc

```diff
@@ -0,0 +1,92 @@
+= "Hibernate OGM 5.4.0.CR1 release"
+Fabio Massimo Ercoli
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+http://hibernate.org/ogm/releases/5.4/#get-it[Hibernate OGM 5.4.0.CR1] has been released!
+
+Here's a list of the main changes:
+
+* We support Infinispan remote transaction over HotRod client
+* Java types `java.time.LocalDate`, `java.time.LocalDateTime` and `java.time.LocalTime` are natively supported as a field types
+* It is possible to configure MongoDB `ReadConcern` strategy
+
+More details available in the
+https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10160&version=31690[release notes].
+
+== Components upgrade
+
+* Hibernate ORM 5.3.4.Final
+* Hibernate Search 5.10.4.Final
+* Infinispan 5.3.3.Final
```

Comment:
Looks like there's a typo here. The Infinispan version seems wrong.

Reply:
There definitely is. Thanks